tracer/test/Datadog.Trace.Security.IntegrationTests/Datadog.Trace.Security.IntegrationTests.csproj

```diff
@@ -51,6 +51,9 @@
     <None Update="remote-rules.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="blocking-ruleset.json">
```

Comment:
No need to change this PR, but I'm not sure it's a good idea to keep adding rule files each time we need a custom rule for testing. I think it probably makes sense to have one file with a custom rule for each scenario we want to test. It can be a future refactoring.

Reply:
true, I changed it, added a dummy rule in ruleset 3.0 so that we can test, fixed in b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba